src/mrpro/data/TrajectoryDescription.py

```diff
@@ -1,29 +0,0 @@
-"""TrajectoryDescription dataclass."""
```

Comment:
I removed this because we are not using it anywhere and I do not see much use in supporting that

Reply:
Isn't this one of the things 8.14 would use?
I thought they store something in these user values?

Reply:
I will ask max next week

Reply:
any update from Max?